agent/src/iconfucius/cli/chat_rasa.py

```
@@ -0,0 +1,382 @@
+                from iconfucius.cli.balance import run_wallet_balance
+                try:
+                    wallet_data = _run_with_spinner(
+                        "Checking wallet...", run_wallet_balance,
+                        ckbtc_minter=False,
+                    )
+                    if wallet_data:
+                        display_text = wallet_data.pop("_display", "")
+                        if display_text:
+                            print(f"{display_text}\n")
+                except Exception:
+                    pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)